packages/ripple-tide-grant/component-loader.js

```diff
@@ -0,0 +1,3 @@
+export default {
```

Comment:
As there are no dynamic components for this content type, we can remove the component loader and component-mapping files